README.md

```diff
@@ -21,6 +21,8 @@ Supports the following platforms:
 
 It also compiles for Android but never been tested on it.
 
+The minimum-supported version fo `rustc` is 1.45 (because of `strip_prefix`).
```

Comment:
```suggestion
The minimum-supported version fo `rustc` is **1.45**.
```

I think the `(because of \`strip_prefix\`)` is "too much". I don't think people are really interested about the why.

Also, I created a topic to add this information in the `Cargo.toml` file directory. If you want to take a look, it's here: https://rust-lang.zulipchat.com/#narrow/stream/246057-t-cargo/topic/minimum.20rust.20version